patch/kernel/rockchip64-dev/add-board-orangepi-4.patch

```diff
@@ -461,8 +462,21 @@ index 00000000..17010305
 +
 +&uart0 {
 +	pinctrl-names = "default";
-+	pinctrl-0 = <&uart0_xfer &uart0_cts>;
++	pinctrl-0 = <&uart0_xfer &uart0_cts &uart0_rts>;
 +	status = "okay";
++	
++	bluetooth {
++		compatible = "brcm,bcm4345c5";
++		clocks = <&rk808 1>;
++		clock-names = "lpo";
++		device-wakeup-gpios = <&gpio2 26 0>; /* GPIO2_D2 GPIO_ACTIVE_HIGH*/
++		host-wakeup-gpios = <&gpio0 4 0>; /* GPIO0_A4 GPIO_ACTIVE_HIGH*/
++		shutdown-gpios = <&gpio0 9 0>; /* GPIO0_B1 GPIO_ACTIVE_HIGH*/
```

Comment:
Use identifiers please - you will not need comments then ;)
```suggestion
+		device-wakeup-gpios = <&gpio2 RK_PD2 GPIO_ACTIVE_HIGH>;
+		host-wakeup-gpios = <&gpio0 RK_PA4 GPIO_ACTIVE_HIGH>;
+		shutdown-gpios = <&gpio0 RK_PB1 GPIO_ACTIVE_HIGH>;
```

Reply:
Thanks,you are right ;)